microcosm_elasticsearch/registry.py

```diff
@@ -0,0 +1,80 @@
+"""
+Manage a set of indexes and/or aliases.
+
+"""
+from elasticsearch_dsl import Index
+from inflection import underscore
+
+
+class IndexRegistry(object):
```

Comment:
No automatic graph binding for the registry? Interested to see how you would bake this in to an app.